apps/web/src/pages/Teams/TeamMembersPage/TeamMembersPage.tsx

```diff
@@ -14,12 +88,114 @@ const TeamMembersPage = () => {
     description: "Page title for the view team members page",
   });
 
+  const columns = React.useMemo<ColumnsOf<TeamMember>>(
+    () => [
+      {
+        Header: intl.formatMessage({
+          defaultMessage: "Actions",
+          id: "OxeGLu",
+          description: "Title displayed for the team table actions column",
+        }),
+        accessor: (d) => `Actions ${d.id}`,
+        Cell: ({ row: { original: member } }: TeamMemberCell) =>
+          actionCell(member, team, roles),
+        disableGlobalFilter: true,
+      },
+      {
+        Header: intl.formatMessage({
+          defaultMessage: "Name",
+          id: "AUOq9D",
+          description:
+            "Title displayed for the team members table Name column.",
+        }),
+        accessor: (d) => getFullNameLabel(d.firstName, d.lastName, intl),
+      },
+      {
+        Header: intl.formatMessage({
+          defaultMessage: "Email",
+          id: "3/lHSy",
+          description:
+            "Title displayed for the team members table email column.",
+        }),
+        accessor: (d) => d.email,
+        Cell: ({ value }: TeamMemberCell) => emailLinkCell(value, intl),
+      },
+      {
+        Header: intl.formatMessage({
+          defaultMessage: "Membership roles",
+          id: "4Washm",
+          description:
+            "Title displayed for the team members table roles column.",
+        }),
+        accessor: (d) => d.id, // TO DO: Update with real role shape
```

Comment:
can this be cleared up now?

Reply:
Old comment 😨 Removed in [remove old comment](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5811/commits/f70fea0e58dbc1b93f48046929a568d8f8e4d263)